packages/sitecore-jss/src/utils/extract-querystring-params.ts

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
I remember refactoring something very similar... layout service and tracking service used to duplicate query string parsing, so we created the 'resolve-url' util in sitecore-jss. Can you please check if that can be reused or updated to cover this use case?

Reply:
We will not be able to use 'resolve-url' as our method does different thing, it extract querystring params from the url.